jupytergraffiti/js/state.js

```diff
@@ -917,20 +918,26 @@ define([
 
     getCellAdditions: () => {
       if (state.history !== undefined) {
-        return state.history.cellAdditions;
+        const  allAdditions = _.union(Object.keys(state.history.cellAdditions), Object.keys(state.playbackCellAdditions));
+        return allAdditions;
       }
       return undefined;
     },
 
-    storeCellAddition: (cellId, position) => {
+    storeCellAddition: (cellId) => {
```

Comment:
this should actually read as it did.

`storeCellAddition: (cellId, position) => {`